TravelExpenses/Proxy Classes/com.sap.expense/internal/TravelexpenseFactory.swift

```diff
@@ -1,20 +1,20 @@
-        TravelexpenseMetadata.EntityTypes.paymentType.registerFactory(ObjectFactory.with(create: { PaymentType(withDefaults: false) }, createWithDecoder: { d in try PaymentType(from: d) }))
-        TravelexpenseMetadata.EntityTypes.reportStatusType.registerFactory(ObjectFactory.with(create: { ReportStatusType(withDefaults: false) }, createWithDecoder: { d in try ReportStatusType(from: d) }))
-        TravelexpenseMetadata.EntityTypes.reservationItemType.registerFactory(ObjectFactory.with(create: { ReservationItemType(withDefaults: false) }, createWithDecoder: { d in try ReservationItemType(from: d) }))
-        TravelexpenseMetadata.EntityTypes.reservationType.registerFactory(ObjectFactory.with(create: { ReservationType(withDefaults: false) }, createWithDecoder: { d in try ReservationType(from: d) }))
-        TravelexpenseMetadata.EntityTypes.tripItemType.registerFactory(ObjectFactory.with(create: { TripItemType(withDefaults: false) }, createWithDecoder: { d in try TripItemType(from: d) }))
+        TravelexpenseMetadata.EntityTypes.currency.registerFactory(ObjectFactory.with(create: { Currency(withDefaults: false) }, createWithDecoder: { d in try Currency(from: d) }))
+        TravelexpenseMetadata.EntityTypes.expense.registerFactory(ObjectFactory.with(create: { Expense(withDefaults: false) }, createWithDecoder: { d in try Expense(from: d) }))
+        TravelexpenseMetadata.EntityTypes.expenseItem.registerFactory(ObjectFactory.with(create: { ExpenseItem(withDefaults: false) }, createWithDecoder: { d in try ExpenseItem(from: d) }))
+        TravelexpenseMetadata.EntityTypes.expenseItemAttachment.registerFactory(ObjectFactory.with(create: { ExpenseItemAttachment(withDefaults: false) }, createWithDecoder: { d in try ExpenseItemAttachment(from: d) }))
+        TravelexpenseMetadata.EntityTypes.expenseReportItem.registerFactory(ObjectFactory.with(create: { ExpenseReportItem(withDefaults: false) }, createWithDecoder: { d in try ExpenseReportItem(from: d) }))
+        TravelexpenseMetadata.EntityTypes.payment.registerFactory(ObjectFactory.with(create: { Payment(withDefaults: false) }, createWithDecoder: { d in try Payment(from: d) }))
+        TravelexpenseMetadata.EntityTypes.reportStatus.registerFactory(ObjectFactory.with(create: { ReportStatus(withDefaults: false) }, createWithDecoder: { d in try ReportStatus(from: d) }))
+        TravelexpenseMetadata.EntityTypes.reservation.registerFactory(ObjectFactory.with(create: { Reservation(withDefaults: false) }, createWithDecoder: { d in try Reservation(from: d) }))
+        TravelexpenseMetadata.EntityTypes.reservationItem.registerFactory(ObjectFactory.with(create: { ReservationItem(withDefaults: false) }, createWithDecoder: { d in try ReservationItem(from: d) }))
+        TravelexpenseMetadata.EntityTypes.tripItem.registerFactory(ObjectFactory.with(create: { TripItem(withDefaults: false) }, createWithDecoder: { d in try TripItem(from: d) }))
```

Comment:
Most of these have incorrect names:
`Expense` → `ExpenseType`
`ExpenseItem` → `Expense`
`ExpenseItemAttachment` → `ExpenseAttachment`
`ExpenseReportItem` → `ExpenseReport`
`Payment` → `PaymentType`
`Reservation` → `ReservationType`
`ReservationItem` → `Reservation`
`TripItem` → `Trip`